doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,98 @@
+"""
+============================================================
+Reconstruct dust-obstructed human cornea image with inpainting
+============================================================
+
+Optical Coherence Tomography (OCT) [1]_ is used to provide eye doctors with an
+image of the retina in the back of a patient's eye.
+
+Quite commonly, there is dust on the reference mirror of the equipment,
+causing dark spots to appear on the acquired images.
```

Comment:
```suggestion
Dust may stick to the reference mirror of the equipment, causing dark spots to
appear on the images. The problem is that these dark spots cover areas of
in-vivo tissue, i.e., they hide data of interest. Our goal here is to
reconstruct the hidden areas based on the pixels near the area boundary.
```

Reply:
@ana42742 @decorouz please :+1: or :-1: the above suggestion.

Reply:
```suggestion
Dust may stick to the reference mirror of the equipment, causing dark spots to
appear on the images. The problem is that these dark spots cover areas of
in-vivo tissue, hence hiding some data of interest. Our goal here is to restore
(reconstruct) the hidden areas based on the pixels near their boundaries.
```
👍